README.md

```diff
@@ -69,6 +69,20 @@ You can download the latest Linux binary from the [release](https://github.com/c
 cy status -o json | jq '.[] | select( .status != "Success")'
 ```
 
+### Create config repository using credential named "Git Config"
+
+```
+GIT_CRED=$(cy --org $ORG credential list -o json | jq '.[] | select( .name == ""Git Config") | .id')
+cy --org myorg  config-repository create --branch master --cred $GIT_CRED --name "lab-config" --default --url "git@github.com:org/repo.git"
+```
+
+### Invite members with "Admin" role
+
+```
+ADMIN_ROLE=$(cy --org $ORG  roles list -o json | jq '.[] | select( .name | contains("Admin")) | .id')
```

Comment:
```suggestion
ADMIN_ROLE=$(cy --org $ORG  roles list -o json | jq '.[] | select( .name == "Admin") | .id')
```

If we're sure it's `Admin` role.

Reply:
Default admin role name is `Organization Admin`. I let "Admin" To put an example with `contains`
But this is not mandatory and maybe make more sense to use `==`

Reply:
Well, it's an example, we can leave it like this in this case :)